app/Models/Bible/BibleFilesetMode.php

```diff
@@ -28,7 +28,7 @@
  * )
  *
  */
-class BibleFilesetSize extends Model
+class BibleFilesetMode extends Model
```

Comment:
It is not related to the current PR but I have fixed it because it was a mistake from the latest PR.